### ISSUE TITLE
Bad command line syntax parsing

### ISSUE THREAD

**Issue by leonidweinbergcx** [Checkmarx/kics#3140]
### Expected Behavior
CLI syntax error thrown

### Actual Behavior
Creates output folder with name "utput-path"

### Steps to Reproduce the Problem

1.  Run command kics scan -output-path ./../ --path ./../project
2.  Program will create utput-path folder in executable directory

### Specifications

-   Version: 1.3.0
-   Platform: Windows
-   Subsystem:

**Comment by rogeriopeixotocx** [comment]
@leonidweinbergcx this issue was reported to cobra CLI parser project https://github.com/spf13/cobra/issues/1399

cc: @felipe-avelar

**Comment by felipe-avelar** [comment]
@leonidweinbergcx I agree with you, I think it should be open until https://github.com/spf13/cobra/issues/1399 and https://github.com/spf13/pflag/issues/314 are solved. It is the same issue, but the problem, probably, is on pflags module, so its need to be fixed on pflag and integrated on cobra...

**Comment by rogeriopeixotocx** [comment]
Yes we will wait for the resolution

**Comment by rafaela-soares** [comment]
Hi, @leonidweinbergcx 🙂 I hope you are doing well.
We are glad to say this bug does not occur anymore. Now, `results.json` is saved in the folder pointed for the user.

![agr](https://user-images.githubusercontent.com/74001161/134651954-106e2971-c67d-47fa-af71-6f306d11291d.PNG)
